squashfs/squashfs_test.go

```diff
@@ -0,0 +1,18 @@
+package squashfs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsEmpytDir(t *testing.T) {
+	t.Parallel()
+	assert := assert.New(t)
+	v, e := isEmptyDir("/tmp")
+	assert.NoError(e)
+	assert.False(v)
```

Comment:
Patch is fine, but technically /tmp certainly can be empty :)  Maybe you should create a file first?

Reply:
Or just check /